dpc-common/src/main/java/gov/cms/dpc/common/hibernate/validator/NoHtmlValidator.java

```diff
@@ -23,7 +23,9 @@ public boolean isValid(String s, ConstraintValidatorContext constraintValidatorC
         if (StringUtils.isBlank(s)) {
             return true;
         }
-        String safe = Jsoup.clean(s, "", Whitelist.none(), new Document.OutputSettings().prettyPrint(false));
-        return safe.equals(s);
+        // Ignore `&` in names and addresses
+        String s1 = s.replaceAll("(\\s&\\s)", "   ");
```

Comment:
Would it make more sense to replace with " and "?
```suggestion
        String s1 = s.replaceAll("(\\s&\\s)", " and ");
```

Reply:
> Would it make more sense to replace with " and "?

I considered this but thought it might be best to maintain the same character length. No one is actually going to see the modified string since the validator just returns a boolean, so I don't think the replacement matters much. I did go back and forth on it for a bit, though. I'm open to changing it.

Reply:
I think you can just change the Whitelist object to whitelist ampersand.

Reply:
@whuang85 i tried that, as i mentioned in the description, but the Whitelist object seems to be made for whitelisting HTML tags and not random strings or regular expressions. I would prefer to use the whitelist object if you can help me figure out how to do that.

Reply:
@jonfulk oops, didn't read the description and yea it looks like it's not possible